src/django_components/context.py

```diff
@@ -0,0 +1,121 @@
+"""
+This file cetralizes various ways we use Django's Context class
```

Comment:
"cetralizes" -> "centralizes"

Reply:
Thanks! Typos fixed in https://github.com/EmilStenstrom/django-components/pull/437/commits/8b8121bde2dd306e2eeed3e5d3ed159d0c666014